endpoints/openrtb2/video_auction.go

```diff
@@ -274,6 +274,17 @@ func (deps *endpointDeps) VideoAuctionEndpoint(w http.ResponseWriter, r *http.Re
 		return
 	}
 
+	ctx = context.WithValue(ctx, exchange.DebugAllowedContextKey, account.DebugAllowed)
+
+	/*if debugLog.Enabled && account.DebugAllowed {
+		debugLog.Data.Request = string(requestJson)
+		if headerBytes, err := json.Marshal(r.Header); err == nil {
+			debugLog.Data.Headers = string(headerBytes)
+		} else {
+			debugLog.Data.Headers = fmt.Sprintf("Unable to marshal headers data: %s", err.Error())
+		}
+	}*/
```

Comment:
Should this be removed? Are the request and header fields added elsewhere?

Reply:
Yes, this commented block of code is moved to line 160. It's not removed.